pom.xml

```diff
@@ -1,26 +1,56 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
 
 	<groupId>de.komoot.photon</groupId>
 	<artifactId>photon</artifactId>
-	<version>0.2.8-SNAPSHOT</version>
+	<version>0.2.9-SNAPSHOT</version>
 
 	<dependencies>
 		<dependency>
 			<groupId>org.elasticsearch</groupId>
 			<artifactId>elasticsearch</artifactId>
-			<version>1.7.0</version>
+			<!-- <version>5.4.1</version> -->
+			<version>5.5.0</version>
+			<exclusions><!-- gives a bad error -->
+				<exclusion>
+					<groupId>org.elasticsearch</groupId>
+					<artifactId>jna</artifactId>
+				</exclusion>
+			</exclusions>
+		</dependency>
+		<dependency><!-- required by elasticsearch -->
+			<groupId>org.elasticsearch.plugin</groupId>
+			<artifactId>transport-netty4-client</artifactId>
+			<!-- <version>5.4.1</version> -->
+			<version>5.5.0</version>
+		</dependency>
+		<dependency><!-- required by elasticsearch -->
+			<groupId>org.apache.logging.log4j</groupId>
+			<artifactId>log4j-api</artifactId>
+			<version>2.7</version>
+		</dependency>
+		<dependency><!-- required by elasticsearch -->
+			<groupId>org.apache.logging.log4j</groupId>
+			<artifactId>log4j-core</artifactId>
+			<version>2.7</version>
 		</dependency>
 		<dependency>
-			<groupId>org.elasticsearch</groupId>
-			<artifactId>elasticsearch-groovy</artifactId>
-			<version>1.7.0</version>
+			<groupId>org.elasticsearch.client</groupId>
+			<artifactId>transport</artifactId>
+			<!-- <version>5.4.1</version> -->
+			<version>5.5.0</version>
 		</dependency>
+		<!-- <dependency> -->
+		<!-- <groupId>org.elasticsearch.module</groupId> -->
+		<!-- <artifactId>lang-groovy</artifactId> -->
+		<!-- <version>2.2.0</version> -->
+		<!-- </dependency> -->
```

Comment:
You are right, I switched to painless (inline) at the end, which is the new ES default scripting language. I first tried the scripting with groovy and started to migrate this by copying also the according lib files to the modules. Nevertheless, groovy is now not required at the end, the last cosmetic finish would be to remove these lines and files accordingly. Waiting for further testing to be finally sure with this.